packages/pegasus/src/ics20.js

```diff
@@ -8,10 +8,15 @@ import { assert, details as X } from '@agoric/assert';
  * https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer#data-structures
  * @property {string} amount The extent of the amount
  * @property {Denom} denom The denomination of the amount
- * @property {string} [sender] The sender address
+ * @property {string} sender The sender address
  * @property {DepositAddress} receiver The receiver deposit address
  */
 
+// ibc-go as late as v3 requires the `sender` to be nonempty, but doesn't
+// actually use it on the receiving side.  We don't need it on the sending side,
+// either, so we can just omit it.
+export const OVERRIDE_SENDER_ADDRESS = 'pegasus';
```

Comment:
not sure it's worth another ci spin, but...

I don't know what we're overriding. "dummy" makes sense to me:

```suggestion
export const DUMMY_SENDER_ADDRESS = 'pegasus';
```